### PR TITLE
Add a test for GetPreamble to ASCIIEncoding

### PR DESCRIPTION
**Pull request by hughbe**
More consistent with other encoding tests

### CODE REVIEW

File: src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetPreamble.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using Xunit;
+
+namespace System.Text.Tests
+{
+    public class ASCIIEncodingGetPreamble
+    {
+        [Fact]
+        public void GetPreamble()
+        {
+            Assert.Equal(new byte[0], new ASCIIEncoding().GetPreamble());
```
Comment (JonHanna) [inline src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetPreamble.cs]: I think I'd find Assert.Empty clearer, but YMMV so don't change this if you find this clearer yourself.

**Comment by JonHanna** [comment]
LGTM

File: src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetPreamble.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetPreamble.cs]: fixed, thanks

**Comment by stephentoub** [comment]
Test Innerloop OSX Release Build and Test please
Test Innerloop Ubuntu Debug Build and Test please

**Comment by stephentoub** [comment]
LGTM

**Pull request merged**